### PR TITLE
[DATASTD-1824] Removed repeated values in descriptor files

### PR DESCRIPTION
**Pull request by mmarable-dlp**

### CODE REVIEW

File: Descriptors/TribalAffiliationDescriptor.xml [hunk 1]
```diff
@@ -3702,12 +3702,6 @@
 		<Description>DEPRECATED: Thlopthlocco Tribal Town</Description>
 		<Namespace>uri://ed-fi.org/TribalAffiliationDescriptor</Namespace>
 	</TribalAffiliationDescriptor>
-	<TribalAffiliationDescriptor>
-		<CodeValue>Tolowa Dee-Ni'</CodeValue>
-		<ShortDescription>DEPRECATED: Tolowa Dee-Ni'</ShortDescription>
```
Comment (vimayya) [inline Descriptors/TribalAffiliationDescriptor.xml:3707]: @mmarable-dlp Do you want DEPRECATED note on the other that is left behind?

File: Descriptors/InternetAccessDescriptor.xml [hunk 2]
```diff
@@ -78,10 +78,4 @@
 		<Description>DEPRECATED: Less Than High Speed</Description>
 		<Namespace>uri://ed-fi.org/InternetAccessDescriptor</Namespace>
 	</InternetAccessDescriptor>
-	<InternetAccessDescriptor>
-		<CodeValue>None</CodeValue>
-		<ShortDescription>DEPRECATED: None</ShortDescription>
```
Comment (vimayya) [inline Descriptors/InternetAccessDescriptor.xml:83]: @mmarable-dlp Do you want DEPRECATED note on the other that is left behind?

File: Descriptors/SpecialEducationSettingDescriptor.xml [hunk 3]
```diff
@@ -90,40 +90,10 @@
 		<Description>Service provider location</Description>
 		<Namespace>uri://ed-fi.org/SpecialEducationSettingDescriptor</Namespace>
 	</SpecialEducationSettingDescriptor>
-	<SpecialEducationSettingDescriptor>
-		<CodeValue>Correctional Facilities</CodeValue>
```
Comment (vimayya) [inline Descriptors/SpecialEducationSettingDescriptor.xml:94]: @mmarable-dlp Do you want DEPRECATED note on the other that is left behind?

Comment (mmarable-dlp) [reply Descriptors/SpecialEducationSettingDescriptor.xml:94]: accdng to this ticket https://tracker.ed-fi.org/browse/DATASTD-1501, Correctional Facilities is one of the values for SpecialEducationSettingDescriptor

**Review by vimayya — APPROVED** [review]

**Pull request merged**